components/lib/inputnumber/InputNumber.js

```diff
@@ -75,11 +76,16 @@ export const InputNumber = React.memo(
             _minusSign.current = getMinusSignExpression();
             _currency.current = getCurrencyExpression();
             _decimal.current = getDecimalExpression();
+            _decimalSeparator.current = getDecimalSeparator();
             _suffix.current = getSuffixExpression();
             _prefix.current = getPrefixExpression();
             _index.current = (d) => index.get(d);
         };
 
+        const getDecimalSeparator = () => {
```

Comment:
OK then can you add a comment in this method exactly what it is doing with examples?

Reply:
Sure,this function is get decimal separator in current locale(`','`,`'.'`)

The reason for these problems is when setting a comma-separated decimal like Germany,the value ​​are like `1231.1` divided by `.`, but the `_decimal .current` is `/[,]/g` 
So what I want is to store a current decimal separator, if the value is a decimal, replace the last `.` with `_decimalSeparator.current`